src/service/tags/tags_repository.py

```diff
@@ -97,3 +99,57 @@ def get_tags(self, offset: int = 0, max_count: int = None) -> list:
         except Exception as error:
             self.logger.error(error)
             return []
+
+    def add_company_tag_query(self, tag_id: str, companies: list) -> str:
```

Comment:
You actually get more than one query. Please rename the method and make it _private_

Reply:
Also, change the name of the function (verb) you're not adding something, you build and get the queries, so it'll be more accurate `get_company_tag_queries`.